dump1090-fa/Dockerfile.template

```diff
@@ -13,6 +13,12 @@ ENV DUMP1090_ADAPTIVE_MIN_GAIN=""
 ENV DUMP1090_ADAPTIVE_MAX_GAIN=""
 ENV DUMP1090_SLOW_CPU=""
 ENV REBOOT_DEVICE_ON_SERVICE_EXIT=""
+ENV AIRSPY_ADSB_SERIAL=""
+ENV AIRSPY_ADSB_BIASTEE="false"
+ENV AIRSPY_ADSB_OPTIONS="-v -t 90 -f 1 -e 4 -w 5 -P 8 -C 60 -E 20 -R rms -D 24,25,26,27,28,29,30,31"
+ENV AIRSPY_ADSB_GAIN="auto"
+ENV AIRSPY_ADSB_SAMPLE_RATE="12"
+ENV AIRSPY_ADSB_STATS="false"
```

Comment:
@ketilmo was there a particular reason you put this in? Does it cause issues putting it to true?

Reply:
AIRSPY_ADSB_STATS false i mean

Reply:
Thanks for the report and feedback, @shawaj! It's great to hear that it looks good so far. I've been running it for a few weeks at one of my sites, and it seems to be working fine, with a significant uptick in reported positions compared to the FlightAware Pro Stick.

I set the logging to false to default to avoid memory card writing.

Reply:
ahh ok - makes sense. I am currently seeing a 30% drop in data compared to radarbox green flight stick. I suspect it is something to do with how I have hooked it up, or maybe it needs some time to stabilise. Not sure but will keep an eye on it

Reply:
Also, I just realised - currently if stats is enabled, it correctly puts stats into the file `/run/airspy_adsb/stats.json`. However this is not exposed outside of the container in any way, and is not fed into the tar1090 container so does not show the stats on the graphs1090 setup.

So we need to add `/run/airspy_adsb` to the lighttpd config in dump1090-fa container, and then also mention about adding these in tar1090 container:

      - ENABLE_AIRSPY=yes
      - URL_AIRSPY=http://airspy_adsb

Reply:
ok i fixed the lighttpd thing and will add the stats enabling info to readme as discussed here https://github.com/ketilmo/balena-ads-b/pull/294#discussion_r2051728064

So will mark this one as resolved now